lambdas/thumbnail/src/t4_lambda_thumbnail/__init__.py

```diff
@@ -90,6 +90,9 @@
 }
 
 
+_TEST_FALLBACK = False
```

Comment:
If this is True, the mode-specific fix I implemented is skipped, and the general fallback fix (likely poor quality) is used.  This is necessary for testing.  I don't like mixing testing logic into my code, but I don't really see an alternative here.